### ISSUE TITLE
我发现，Linux的包没有图标

### ISSUE THREAD

**Issue by dongasai** [kingToolbox/WindTerm#320]
没有图标我从源代码里下载了一个图标，用来建立快捷方式

**Comment by kingToolbox** [comment]
Yes, this is just a **portable** software, so there is no `desktop` file, Because I cannot know which directory you put `WindTerm` in, I cannot configure the desktop entries `Exec` and `Icon` in advance. if you need it, please create it manually. If you have a better way, please let me know. Thank you.

**Comment by dongasai** [comment]
编写一个脚本来适应linux的多种桌面环境的规则是一个不错的选择，但是成本较高；使用Linux的都是具有一定经验的开发者，所以我认为比较简单的方案是：在Linux的包中添加图标文件，我想大部分使用者都是可以自己创建快捷方式的，我遇到的问题就是缺少一个创建快捷方式的图标文件。

**Comment by kingToolbox** [comment]
I am fully agree with you. I will place an 1024*1024 icon file in the next `Linux` version package to make it easy for users who need it. Thank you very much.

**Comment by kingToolbox** [comment]
I am happy to inform you that [WindTerm_2.3.0](https://github.com/kingToolbox/WindTerm/releases/tag/2.3.0) has been released and a `WindTerm.png` and a `WindTerm.desktop` have been added to the `Linux` version, please download and check it, thank you.

BTW:
The `WindTerm.desktop` created by `sakura1943` from [archlinux](https://aur.archlinux.org/packages/windterm-bin), You can customise it to suit your needs.
